contrib/coreos/user-data.example

```diff
@@ -133,3 +134,14 @@ write_files:
       TOOLBOX_DOCKER_IMAGE=ubuntu-debootstrap
       TOOLBOX_DOCKER_TAG=14.04
       TOOLBOX_USER=root
+  - path: /etc/environment_proxy
```

Comment:
In my installation I use /etc/environment because it does make sense to use for the whole system

but it is up to you